upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -691,7 +691,21 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 				Version:           fi.String(versions[id]),
 				Selector:          networkingSelector,
 				Manifest:          fi.String(location),
-				KubernetesVersion: ">=1.8.0",
+				KubernetesVersion: ">=1.8.0 <1.9.0",
```

Comment:
As mentioned, change to `>=1.8.0 <1.9.3` and then the `k8s-1.9` block below to apply on Kubernetes `>=1.9.3`

Reply:
> actually dropping the check on the ipvs mode as it feels hacky with the check in validation due to blocking, or locking the version to 1.9.3, as they might not be using ipvs anyhow. I'm gonna leave it for another PR or perhaps leave kube admins to decide for now

^ I added another commit and removed the check for now .. feels weird using <1.9.3 as stated above. I'll raise an issue and see what they decide .. for now, i'll just keep the PR to adding canal v3 support.

Reply:
Post a 3.2.3 update .. rules seem to be fine ..
```shell
core@ip-10-250-32-234 ~ $ sudo iptables -L FORWARD -nv
Chain FORWARD (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 KUBE-FORWARD  all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* kubernetes forwarding rules */
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            169.254.169.254      tcp dpt:80 state NEW /* AWS metadata endpoint */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:22 state NEW /* SSH */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:2380:2381 state NEW /* ETCD Peering */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:4001:4002 state NEW /* ETCD API */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:4194 state NEW /* cAdvisor */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:8989 state NEW /* Kube Auth Listener */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:9091 state NEW /* Calico Felix Prometheus Metrics */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpt:9099 state NEW /* Calico health check */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.10.0.0/16         tcp dpts:10250:10256 state NEW /* kube http services */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:22 state NEW /* SSH */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:2380:2381 state NEW /* ETCD Peering */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:4001:4002 state NEW /* ETCD API */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:4194 state NEW /* cAdvisor */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:8989 state NEW /* Kube Auth Listener */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:9091 state NEW /* Calico Felix Prometheus Metrics */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpt:9099 state NEW /* Calico health check */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:10250:10256 state NEW /* kube http services */ reject-with icmp-port-unreachable
    0     0 REJECT     tcp  --  cali+  *       0.0.0.0/0            10.250.0.0/16        tcp dpts:30000:32767 state NEW /* nodeport range */ reject-with icmp-port-unreachable
    0     0 ACCEPT     all  --  *      *       10.10.0.0/16         0.0.0.0/0           
    0     0 ACCEPT     all  --  *      *       0.0.0.0/0            10.10.0.0/16        
    0     0 cali-FORWARD  all  --  *      *       0.0.0.0/0            0.0.0.0/0            /* cali:wUHhoiAYhphO9Mso */
```

Reply:
The issue is that the cali-FORWARD rule is appended at the bottom, after the ACCEPT rules added by flannel. So any k8s network policies won't take effect with this setup.